srcpkgs/alsa-ucm-conf/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'alsa-ucm-conf'
+pkgname=alsa-ucm-conf
+version=1.2.4
+revision=1
+archs="i686* x86_64*"
+depends="alsa-lib"
+short_desc="ALSA Use Case Manager topology configurations"
+maintainer="cinerea0 <cinerea0@protonmail.com>"
+license="BSD-3-Clause"
+homepage="https://github.com/alsa-project/alsa-ucm-conf"
+distfiles="https://www.alsa-project.org/files/pub/lib/${pkgname}-${version}.tar.bz2"
+checksum=2c3b535c77dcb9aaf62a61f4f8324f1ab184162f105f7ec9ed1e37c742fcd340
+
+do_install() {
+	vinstall ucm2/README.md 0755 /usr/share/doc/${pkgname}
```

Comment:
For that matter, this should probably be `vdoc`.

And you don't want to use 755 permissions for documentation :P

Reply:
Would you believe I forgot `vdoc` exists? I've changed the line.